todo-backend/README.adoc

```diff
@@ -405,10 +509,20 @@ endif::[]
 The backend will be built and deployed on OpenShift with a Helm Chart for {productName}.
 
 ifndef::ProductRelease,EAPXPRelease[]
+Add the bitnami repository which provides an helm chart for PostgreSQL:
+[source,options="nowrap"]
+----
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+"bitnami" has been added to your repositories
+----
+
+Install the full application (database + backend).
+
 [source,options="nowrap",subs="+attributes"]
 ----
-$ helm install todo-backend --set build.ref={WildFlyQuickStartRepoTag} --set build.s2i.version={productImageVersion} -f https://raw.githubusercontent.com/wildfly/wildfly-charts/main/examples/todo-backend/todo-backend-s2i.yaml wildfly/wildfly
-NAME: todo-backend
+$ helm dependency update todo-backend-chart/
+$ helm install todo --set build.ref={WildFlyQuickStartRepoTag} todo-backend-chart
```

Comment:
`--set build.ref` should be removed, other QS don't use any custom set Git repo references.

Reply:
It's OK now, thanks